guvnor-services/guvnor-services-backend/src/main/java/org/guvnor/common/services/backend/metadata/attribute/GeneratedAttributesView.java

```diff
@@ -98,7 +98,13 @@ public Object fileKey() {
     private boolean extractGenerated() {
         final Map<String, Object> content = path.getAttrStorage().getContent();
 
-        return Boolean.parseBoolean( String.valueOf( content.get( GENERATED_ATTRIBUTE_NAME ) ) );
+        final Object generatedFileAttribute = content.get(GENERATED_ATTRIBUTE_NAME);
+
+        if (generatedFileAttribute instanceof Boolean) {
```

Comment:
Just a defensive programming in case someone inserts a non-Boolean value for the key. Moreover the instanceof provides a null check for free.